api/Trace/Span.php

```diff
@@ -9,8 +9,8 @@
 interface Span extends SpanStatus, SpanKind
 {
     public function getSpanName(): string;
-    public function getContext(): SpanContext;
-    public function getParent(): ?SpanContext;
+    public function getContext(): Baggage;
```

Comment:
Should this function name change too?

Reply:
I think we could change it to `getBaggage`.

Reply:
Thinking about this statement more, there is a difference between `Context` and SpanContext`.  Perhaps it's smart to just change `SpanContext`?

Reply:
Good call.  Still says getContext in the spec :)